testsuite/regressiontests/issue-410.sli

```diff
@@ -0,0 +1,118 @@
+/*
+ *  issue-410.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+/* BeginDocumentation
+   Name: testsuite::issue-410 - trigger update weight fix
+
+   Synopsis: (issue-410) run -> NEST exits if test fails
+   Description:
+   This test compares the impact of the volume transmitter
+   on a dopamine synapse using different number of threads.   
+
+   FirstVersion: 07/2016
+   Author: Till Schumann
+ */
+
+% The following test needs the model iaf_cond_alpha, so
+% this test should only run if we have GSL
+statusdict/have_gsl :: not {statusdict/exitcodes/success :: quit_i} if
+
+
+(unittest) run
+/unittest using
+
+M_ERROR setverbosity
+
+is_threaded_with_openmp not { exit_test_gracefully } if
+
+/sim_dopa_concentration
```

Comment:
Minor comment: This function returns the synaptic weight but the name of the function implies that you return the dopamine concentration.